src/cephadm/cephadm

```diff
@@ -5792,14 +5792,12 @@ def check_time_sync(ctx, enabler=None):
 
 
 def command_check_host(ctx: CephadmContext) -> None:
-    container_path = ctx.container_engine.path
-
     errors = []
     commands = ['systemctl', 'lvcreate']
 
     try:
-        check_container_engine(ctx)
-        logger.info('podman|docker (%s) is present' % container_path)
+        engine = check_container_engine(ctx)
+        logger.info('podman|docker (%s) is present' % engine.path)
```

Comment:
while you're here: could you please also print the podman version?

Reply:
`podman (/usr/bin/podman) version 3.2.3 is present` ?